### ISSUE TITLE
update footer links

### ISSUE THREAD

**Issue by alishaevn** [scientist-softserv/phenovista-digital-storefront#15]
# Acceptance Criteria
- [ ] the footer links are updated to reflect the request below
- [ ] the additional legal policies that are currently in the footer can be removed as they'll be placed somewhere else

# Story
from the client:
> 3 columns (can we have all links open in a new tab?):
>
> Service Offerings
Imaging & Analysis (https://phenovista.com/imaging-analysis)
Ready-2-Go Assays (https://phenovista.com/r2g-assay-services)
Cell Painting & Screening (https://phenovista.com/cell-painting-morphology-assay-services)
Bespoke Services (https://phenovista.com/bespoke-assay-services)
>
> Research Areas
Fibrosis (https://phenovista.com/fibrosis-assay-services)
Neurobiology (https://phenovista.com/neurobiology-assay-services)
Oncology & Immuno-oncology (https://phenovista.com/oncology-immuno-oncology-assay-services)
Cell & Gene Therapy (https://phenovista.com/cell-and-gene-therapy-assay-services)
>
> Helpful Links
About Us (https://phenovista.com/about-us)
Expertise (https://phenovista.com/expertise)
Privacy Policy

**Comment by crisr15** [comment]
Internal QA failures: 

- [ ] It was requested that the footer links open in a new tab.  They are opening in the current tab. 
- [ ] The LinkedIn icon is not showing on the footer. Add linkedin (https://www.linkedin.com/company/phenovista-biosciences/)

**Comment by alishaevn** [comment]
the linked in icon is not set up for use in the footer at the moment. it will be updated with https://github.com/scientist-softserv/webstore-component-library/issues/194.

the links are still opening in the same tab however when they were requested to open in a new tab. pulled back to "ready" and labeled as "needs rework".

**Comment by summer-cook** [comment]
picking back up to make links open in a new tab

**Comment by summer-cook** [comment]
so i just looked at this for a few mins, in order to make them open in a new tab, it would need to be a change to the component library itself. should i make it an option that is passed in to the links from the webstore?

ideally we would want only external links to open in a new tab and internal (webstore) links to open in the same tab. or we would pass something in the FOOTER_SECTIONS object saying whether the link is internal or external

**Comment by alishaevn** [comment]
I agree with your assessment. let's pass a "target" prop to the footer that can be any of the values that the attribute receives. handling it this way will prepare us for the possibility of using other link attributes in the future.

**Comment by summer-cook** [comment]
Note: the work for this in phenovista is done, but we still need to do a wcl release, update the webstore with the new wcl, and then pull that most recent version into the phenovista repo as well. 

We are waiting until 
- https://github.com/scientist-softserv/webstore-component-library/pull/195
gets merged to be able to do one single release for the wcl with all this work
